packages/rum-core/src/boot/rumPublicApi.ts

```diff
@@ -194,37 +397,25 @@ export function makeRumPublicApi(startRumImpl: StartRum, recorderApi: RecorderAp
   const rumPublicApi = makePublicApi({
     init: monitor((initConfiguration: RumInitConfiguration) => strategy.init(initConfiguration)),
```

Comment:
**💬 suggestion:** ‏With a little bit of tweaking in `makePublicApi`, we could avoid duplicating a bit of typings here.

```ts
export function makePublicApi<T extends PublicApi>(stub: Omit<T, keyof PublicApi>): T {
   ...
  return publicApi as T
}
```

then, no need to specify parameters types anymore:

```suggestion
  const rumPublicApi = makePublicApi({
    init: monitor((initConfiguration) => strategy.init(initConfiguration)),
```